Jellyfin.Server/Migrations/Routines/MigrateLibraryDb.cs

```diff
@@ -94,7 +94,7 @@ public void Perform()
         connection.Open();
 
         var baseItemIds = new HashSet<Guid>();
-        using (var operation = GetPreparedDbContext("moving TypedBaseItem"))
+        using (var operation = GetPreparedDbContext("Moving TypedBaseItem"))
```

Comment:
the log texts here are concatinated so they are not the start of the sentence and lower case is correct here